### DISCUSSION TITLE
也算是吃上乔普师傅的面了

### DISCUSSION THREAD

**Discussion by cpu644** [share121/inter-knot#1289] [其他]
虽然贵 但是这丁尼没白花！吃完甚至感觉强壮了一些。可惜的是师傅说他刚订的蛋/海苔被物流丢空洞里了... 哎。可惜了，下次再来吧。
<img width="3998" alt="Noodle1" src="https://github.com/user-attachments/assets/194410bb-918b-42e5-bd8a-42b8c39b7508">
<img width="2376" alt="Noodle2" src="https://github.com/user-attachments/assets/41e7f598-1c50-450c-9e85-971465989680">

**Comment by suiry-D** [comment]
丢空洞了？这个委托我接了！

**Comment by ZhuYuan-148** [comment]
![mmexport1723351278984](https://github.com/user-attachments/assets/8f391caa-2b75-4bd8-8e27-d9e08da903d9)
上次深夜执勤去吃的，乔普大叔人很好！

**Reply by cpu644** [reply]
看饿了，明天继续去吃！1！！1！
